modules/cluster/main.tf

```diff
@@ -67,6 +67,34 @@ module "eks" {
   subnets         = (var.cluster_in_private_subnet ? module.vpc.private_subnets : module.vpc.public_subnets)
   vpc_id          = module.vpc.vpc_id
   enable_irsa     = true
+
+  worker_groups_launch_template = var.enable_worker_groups_launch_template ? [
+    for subnet in module.vpc.public_subnets :
```

Comment:
Proabably have to do something similar for private subnets, ignore for this PR ...